packages/kbn-cli-dev-mode/src/dev_server.test.ts

```diff
@@ -123,29 +123,10 @@ describe('#run$', () => {
     // ensure that FORCE_COLOR is in the env for consistency in snapshot
     process.env.FORCE_COLOR = process.env.FORCE_COLOR || 'true';
 
-    expect(execa.node.mock.calls).toMatchInlineSnapshot(`
-      Array [
-        Array [
-          "some/script",
-          Array [
-            "foo",
-            "bar",
-          ],
-          Object {
-            "env": Object {
-              "<inheritted process.env>": true,
-              "ELASTIC_APM_SERVICE_NAME": "kibana",
-              "isDevCliChild": "true",
-            },
-            "nodeOptions": Array [
-              "--preserve-symlinks-main",
-              "--preserve-symlinks",
-            ],
-            "stdio": "pipe",
-          },
-        ],
-      ]
-    `);
+    const [script, args, options] = execa.node.mock.calls[0];
```

Comment:
This used to test ENV, node options, and that only one call was made, why change it?

Reply:
I can add a check that makes sure there is only one call. It included the node env, which differs if you're running from `scripts/jest` or if we are calling directly with `node_modules/.bin/jest`

Reply:
The inherited node env should be captured under `<inheritted process.env>`, including any unique env vars passed to the script. Mind if I try to keep the snapshot in place?

Reply:
Validating the env and the node options provided for the script is definitely intentional

Reply:
Yeah, go for it

Reply:
Ah, ran it locally and it's the `nodeOptions` which are inheritted and not set when running via `node_modules/.bin/jest`.